app/styles/pages/subtopics.scss

```diff
@@ -0,0 +1,3 @@
+#subTopicPlaceholder {
```

Comment:
This is not needed. Semantic ui already has a class for responsive images. Use that please.